### PR TITLE
added heading in html file 

### PR DESCRIPTION
**Pull request by abdulrazzaq123**
please accept my contribution to this project I have added new features I have worked really hard

### CODE REVIEW

File: hgf.html [hunk 1]
```diff
@@ -9,6 +9,7 @@
     <script src='main.js'></script>
 </head>
 <body>
-    kghb
+    <h1>hii</h1>
```
Comment (MSR806) [inline hgf.html]: This is very unprofessional please change this and add `<h1>Hi sir!</h1>`

**Review by MSR806 — CHANGES_REQUESTED** [review]
I have reviewed your code.
have a look at the comments and update accordingly
Thank you

**Comment by abdulrazzaq123** [comment]
> I have reviewed your code.
> have a look at the comments and update accordingly
> Thank you

I will update the code and get back to you 
thanks for the review

**Comment by MSR806** [comment]
Very good @abdulrazzaq123  you have done a brilliant job in here. I will accept this code.
Thank you for your contribution.

**Pull request merged**